relay/components/Releases/PublishReleaseForm.tsx

```diff
@@ -91,6 +93,9 @@ export const PublishReleaseForm:FunctionComponent<any> = ({ orgName, repoName }:
           const metaResp = await valist.getRepoMeta(orgName, repoName);
           console.log(metaResp);
           setRepoMeta(metaResp);
+
+          const canRelease = await valist.canRelease(orgName, repoName);
```

Comment:
When the build runs it triggers the linting and fails on this line because `canRelease` is already defined as the state value and the linting prevents shadowing names from parent scope. Maybe the internal value can be renamed to something like `userCanRelease`. Forgot to mention that we tend to run `make build` to test the production linting.